src/components/AnalyticsOptInModal.vue

```diff
@@ -46,19 +82,20 @@ export default {
     ...mapActions(['setAnalyticsResponse', 'initializeAnalytics']),
     close() {
       this.setAnalyticsOptInModalOpen({ open: false })
+      this.$emit('goToSetup')
     },
     async accept() {
       this.loading = true
+      this.analyticsAccepted = true
```

Comment:
Not need it if you use the acceptedDate from the state

Reply:
> Not need it if you use the acceptedDate from the state

Fixed but on `reject`, the state value is `null`. Have to add a data property for accepting. Pushed a fix. Kindly review.